netmiko/cisco/cisco_vxr_ssh.py

```diff
@@ -543,7 +547,10 @@ def commit(self, confirm=False, confirm_delay=None, comment='', label='', replac
                 msg = "Prompt not found after sending commit replace command"
                 self.log.error(msg)
                 raise PatternNotFoundException(msg)
-            expect_string = r'\)#$' + "|" + alt_error_marker
+            if self.base_prompt.startswith("sysadmin-vm:"):
```

Comment:
better to store in a boolean flag then checking this 3-4 times in the same method or store the expect string once and reuse same variable everywhere (append with alt_error_marker or other expected strings in special cases)

Reply:
Good idea. For now, I will leave it like this as the code logic works and has been tested with the below logs. 
The variable value deduplication will be tried to address in next PR.